knative-operator/pkg/controller/knativekafka/rbacproxytransforms.go

```diff
@@ -0,0 +1,50 @@
+package knativekafka
+
+import (
+	"context"
+	"errors"
+
+	mf "github.com/manifestival/manifestival"
+	operatorv1alpha1 "github.com/openshift-knative/serverless-operator/knative-operator/pkg/apis/operator/v1alpha1"
+	"github.com/openshift-knative/serverless-operator/openshift-knative-operator/pkg/monitoring"
+	"k8s.io/apimachinery/pkg/util/sets"
+	eventingv1alpha1 "knative.dev/operator/pkg/apis/operator/v1alpha1"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+var (
+	kafkaChannelComponents = []string{"kafka-ch-controller", "kafka-ch-dispatcher", "kafka-webhook"}
+	kafkaSourceComponents  = []string{"kafka-controller-manager"}
+)
+
+func addRBACProxySupportToManifest(instance *operatorv1alpha1.KnativeKafka, components []string) (*mf.Manifest, error) {
+	proxyManifest := mf.Manifest{}
```

Comment:
for eventing, I see https://github.com/openshift-knative/serverless-operator/blob/6f9d2ca6d6a8a61b251315d2d326b330094dcdca/openshift-knative-operator/pkg/monitoring/helpers.go#L260  is called ?

Reply:
Yes for example `GetEventingMonitoringPlatformManifests` calls that. We dont need to do so for Kafka because the Prometheus account in `openshift-monitoring` ns already gets the proper rbac roles due to the GetEventingMonitoringPlatformManifests call.